eslint.config.mjs

```diff
@@ -0,0 +1,18 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+export default [
+  {
+    files: ["**/*.js"],
+    languageOptions: {
+      sourceType: "script",
+      globals: globals.browser
+    }
+  },
+  {
+    ignores: ['dist/']
```

Comment:
We need to exclude `/public` too as this contains generated files which we don't have lots of control over:

```suggestion
    ignores: ['dist/', 'public/']
```